common.cc

```diff
@@ -346,7 +346,9 @@ void connect_agent(void) {
     string host = string("tcp://") + string(args.agent_arg[i]) +
       string(":") + string(args.agent_port_arg);
     s->connect(host.c_str());
-    s->setsockopt(ZMQ_RCVTIMEO, 10000);
+    int64_t timeout = 10000;
```

Comment:
```suggestion
    int timeout = 10000;
```

The parameter must be int. It is checked by zmq.